packages/cactus-plugin-ledger-connector-aries/src/main/typescript/generated/openapi/typescript-axios/git_push.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
@outSH Existing patterns we have in place exclude this file and the npmignore file and some other one that I can't remember right now (I thiknk the .gitignore). Could you please take a look at the other packages on how to handle the open api ignore files? I think Jagpreet recently made some updates so that the open api ignore file is somehow centralized (which allows us to delete the generated sources and regenerate them without having to create changes in the files that are under version control - or at least this how i remember it in terms of the big picture)

Reply:
Yeah, sorry, I always forget about that - fixed

Reply:
@outSH No worries! It's one of those annoying chores that we should have some automation to check for in the `custom-checks` script.